buildscripts/kokoro/cronet.sh

```diff
@@ -3,6 +3,119 @@
 set -exu -o pipefail
 cat /VERSION
 
-cd ./github/grpc-java/cronet
+BASE_DIR=$(pwd)
+
+# Set up APK size and dex count statuses
+
+gsutil cp gs://grpc-testing-secrets/github_credentials/oauth_token.txt ~/
+
+function set_status_to_fail_on_error {
+  cd $BASE_DIR/github/grpc-java
+
+  # TODO(ericgribkoff) Remove once merged
+  git checkout $KOKORO_GITHUB_PULL_REQUEST_COMMIT
```

Comment:
This seems unfortunate. I don't think it's something I'd want us to remove (rather, I want us to always use the most recent version of the script.) Maybe there's an alternative, like copying the script some place else...